src/index.css

```diff
@@ -10,5 +22,182 @@
   display: flex;
   flex-direction: row;
   width: 100%;
+  background-color: #fff;
+  border-radius: 5px;
+  border-color: #e8e8e8;
+  border-width: 1px;
+  border-style: solid;
+  height: 22vh;
+  margin-top: 2.375rem;
+  padding: 20px;
+}
+
+.category {
+  font-family: 'Montserrat', sans-serif;
+  color: #121212;
+  margin: 0 2.438rem 0 0;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.title {
+  font-size: 1.375rem;
+  width: 20vw;
+  font-family: 'Roboto Slab', serif;
+  color: #121212;
+}
+
+.author {
+  margin: 0.25rem 0.875rem 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.interact {
+  margin-top: 5%;
+}
+
+.interact-btn {
+  background-color: rgba(1, 1, 1, 0);
+  border: none;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  color: #4386bf;
+}
+
+.sep {
+  border-left-style: solid;
+  border-left-color: #e8e8e8;
+  border-left-width: 1px;
+}
+
+.left {
+  width: 25rem;
+}
+
+.middle {
+  display: flex;
+  margin-top: 0.3rem;
+}
+
+.progress-wheel {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+  border-color: #379cf6;
+  border-radius: 50%;
+  border-image-slice: 1;
+}
+
+.percentage {
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', serif;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.line {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
```

Comment:
- [ ] I think too much margin in this line is pushing the right element out o the viewport. I suggest you use relative units preferably `%` for the width of the `list-item` children elements. The issue of overflowing elements has been mentioned by the previous reviewer. :pray: 

![image](https://user-images.githubusercontent.com/64894927/135672293-102fc9cf-4f60-42ab-af9e-3d9bfb9b4a76.png)